google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/datacatalog_tag_factory.py

```diff
@@ -104,6 +104,43 @@ def make_tag_for_table_metadata(self, tag_template, table,
 
         return tag
 
+    def make_tags_for_columns_metadata(self, tag_template, table):
+        """
+         Create Tags for the Table Columns technical fields that
+          aren't support yet by Datacatalog api.
+
+         :param tag_template: A datacatalog.TagTemplate()
+         :param table:
+         :return: list[tag]
+        """
+
+        tags = []
+
+        columns = table.get('columns')
+
+        if columns:
+            for column in columns:
+                tag = datacatalog.Tag()
+
+                tag.template = tag_template.name
+
+                mask = column.get('mask')
```

Comment:
Aren't the `mask` and `mask_expression` fields specific to SAP Hana columns? If so, maybe it is better to handle them in the SAP Hana connector. WDYT? 🤔

Reply:
Right now yes, but they are generic fields not tied to SAP Hana.

To use it specifically in SAP Hana it will take a lot more code, because the connector would need to extend the tag  template factory class. So I'd like to avoid this.